$NetBSD: patch-aa,v 1.1 2010/06/14 06:49:51 wiz Exp $

Conflicts with built-in getline() function on some platforms, and
is not even used in the file.

--- src/gui-none/nogui.c.orig	2004-09-07 14:11:55.000000000 +0000
+++ src/gui-none/nogui.c
@@ -80,10 +80,6 @@ void gui_filename (int num, const char *
 {
 }
 
-static void getline (char *p)
-{
-}
-
 void gui_handle_events (void)
 {
 }
